server/api.py

```diff
@@ -286,3 +285,21 @@ def create_session(claims, token_user):
 
         return jsonify(success="Successfully created session", session=new_session), 200, json_tag
 
+    def read_session(claims, token_user, id):
+        """ Returns the session with the given id """
+        user = None
+        try:
+            user = token_user
+        except:
+            return malformed_request()      
+
+        # Make sure the session exists
+        try:
+            remote_copy = Session.objects(id=id).first()
```

Comment:
is there any specific reason that we are naming all of these remote_copy?

Reply:
I have no idea, I was just staying consistent with the rest of the file